packages/legacy/core/App/components/animated/RecordLoading.tsx

```diff
@@ -1,55 +1,72 @@
 import React, { useEffect, useRef } from 'react'
-import { View, StyleSheet, Animated } from 'react-native'
+import { StyleProp, ViewStyle, View, StyleSheet, Animated } from 'react-native'
 
 import { useTheme } from '../../contexts/theme'
 import { testIdWithKey } from '../../utils/testable'
 
+type RecordLoadingProps = {
+  style?: StyleProp<ViewStyle>
+}
+
 const fadeTiming: Animated.TimingAnimationConfig = {
-  toValue: 0.2,
+  toValue: 0.4,
   duration: 1100,
   useNativeDriver: true,
 }
 
-const RecordLoading: React.FC = () => {
+const RecordLoading: React.FC<RecordLoadingProps> = ({ style }) => {
   const { ColorPallet } = useTheme()
   const rowFadeAnim = useRef(new Animated.Value(1))
-  const style = StyleSheet.create({
+  const myStyle = StyleSheet.create({
     container: {
       flexDirection: 'column',
     },
     rectangle: {
-      backgroundColor: ColorPallet.grayscale.veryLightGrey,
-      height: 30,
-      marginVertical: 10,
-    },
-    line: {
       backgroundColor: ColorPallet.grayscale.lightGrey,
-      height: 1,
+      height: 30,
       marginVertical: 5,
+      borderRadius: 10,
     },
   })
 
   useEffect(() => {
     Animated.loop(Animated.timing(rowFadeAnim.current, fadeTiming)).start()
   }, [])
 
-  const makeARow = () => {
-    return (
-      <View style={{ flexDirection: 'column' }}>
-        <View style={[style.rectangle, { width: '35%' }]}></View>
-        <View style={[style.rectangle, { width: '85%' }]}></View>
-        <View style={style.line} />
-      </View>
-    )
-  }
-
   return (
-    <View style={style.container} testID={testIdWithKey('RecordLoading')}>
-      <Animated.View style={{ opacity: rowFadeAnim.current }}>
-        {makeARow()}
-        {makeARow()}
-      </Animated.View>
-    </View>
+    <Animated.View
+      style={[{ opacity: rowFadeAnim.current, backgroundColor: ColorPallet.grayscale.white, borderRadius: 15 }, style]}
+    >
+      <View style={myStyle.container} testID={testIdWithKey('RecordLoading')}>
+        <View style={{ flexDirection: 'row' }}>
+          <View
+            style={{
+              marginLeft: 0,
+              backgroundColor: ColorPallet.grayscale.lightGrey,
+              width: 40,
+              borderTopLeftRadius: 15,
+              borderBottomLeftRadius: 15,
+            }}
+          />
+          <View
+            style={{
+              marginLeft: -22,
+              marginTop: 20,
+              backgroundColor: ColorPallet.grayscale.lightGrey,
+              height: 45,
+              width: 45,
+              borderRadius: 5,
```

Comment:
These width / margin numbers seem somewhat arbitrary, could they be explained or named?